src/components/FruitDisplays/index.js

```diff
@@ -0,0 +1,86 @@
+import React from "react"
+import "src/styles/tailwind.css"
+import { Segmented, Space, Button } from "antd"
+import { CloseOutlined, EditOutlined } from "@ant-design/icons"
+
+function FruitDisplays({ id, names, prices, onDelete, onEditButtonClick }) {
+  return (
+    <div style={{ "margin-right": "4%" }}>
+      <div
+        style={{
+          "margin-right": "2%",
+          "margin-left": "2%",
```

Comment:
try avoid to use percentage for styling, maybe use some in order to limit size of something, at most cases you should use "px" (pixels) or "em"/"em" for styling for expected styling.